### DISCUSSION TITLE
Verify error: StatusCode: 400 when trying to verify number for use with signal-cli

### DISCUSSION THREAD

**Discussion by leehendo** [AsamK/signal-cli#1615] [Q&A]
Hi,
Followed the Quickstart guide for registering a number and have the captcha token and applied it using the correct format.
Verify code sent to my mobile, but when using the below:

./signal-cli -u +447............. verify 123456

I get Verify error: StatusCode: 400 

what am I doing wrong?

:)

**Comment by AsamK** [comment]
See https://github.com/AsamK/signal-cli/issues/1614
The fix will be released to soon

**Reply by leehendo** [reply]
Thank you sir.

**Comment by leehendo** [comment]
Thank you.

 

Can I be notified?

 

Best Regards

Lee Henderson

Mobile                       07710 414583
Direct Dial                 01580 200700
Website                    http://www.4productions.co.uk


NOTE: This message and any attached files may contain information that is confidential and/or subject of legal privilege.
If you are not the intended recipient or the person responsible for delivering the message, be advised that you have
received this message in error and that any dissemination, copying or use of this message or attachment is forbidden,
as is the disclosure of the information therein.  If you have received this message in error please notify the sender
immediately and delete the message. This communication and its contents are transmitted without prejudice and subject
to contract.

 

From: Sebastian Scheibner ***@***.***>
Reply to: AsamK/signal-cli ***@***.***>
Date: Monday, 28 October 2024 at 16:48
To: AsamK/signal-cli ***@***.***>
Cc: leehendo ***@***.***>, Author ***@***.***>
Subject: Re: [AsamK/signal-cli] Verify error: StatusCode: 400 when trying to verify number for use with signal-cli (Discussion #1615)

 

See #1614
The fix will be released to soon
